test/test_examples.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class TestScarpe < Minitest::Test
+  def test_examples
+    examples_to_test = Dir["examples/**/*.rb"]
+      .reject { _1.include?("/not_checked/") }
+      .reject { _1.include?("/skip_ci/") if ENV["CI_RUN"] }
```

Comment:
I tried for a while and it seemed like negative match is not a thing for globs, but maybe someone knows a more clever approach.

Reply:
You can do everything-glob, call .to_a, and then subtract the matches you don't want. But it's not cleaner than this, and I wouldn't use it.